Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -67,34 +76,63 @@ public partial class GestureManager : Singleton<GestureManager>
         public Vector3 ManipulationOffset { get; private set; }
 
         /// <summary>
-        /// The world space position of the hand being used for the current manipulation gesture.  Not valid
-        /// if a manipulation gesture is not in progress.
+        /// The world space position of manipulation source being used for the current manipulation gesture.
```

Comment:
I used the all the scenes in the test folder to test functionality.  

Haven't tested with clicker yet. I was hoping to have done that last week, but Hurricanes always ruin the party. haha.